testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractBaseBrokerTest.java

```diff
@@ -280,6 +280,9 @@ protected String getLoginUrl(String contextRoot, String realmName, String client
         assertThat(clients, Matchers.is(Matchers.not(Matchers.empty())));
 
         String redirectURI = clients.get(0).getBaseUrl();
+        if (redirectURI.startsWith("/")) {
```

Comment:
this was necessary because the generated redirect URI for the "account" client was "/realms/account" ...